WCFDataService/Client/System/Data/Services/Client/Common/DataServiceKeyAttribute.cs

```diff
@@ -28,7 +28,7 @@ namespace System.Data.Services.Common
 
     /// <summary>Denotes the key property or properties of an entity. </summary>
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1019:DefineAccessorsForAttributeArguments", Justification = "Accessors are available for processed input.")]
-    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface, AllowMultiple = false)]

```

Comment:
Is there a test we can add to ensure that the attribute is affecting interfaces correctly?

Reply:
I havent found any tests in older branch - do they even exists ?